src/components/events/createEvent/EventsAdditionalInfo.jsx

```diff
@@ -5,7 +5,7 @@ import { RightOutlined } from '@ant-design/icons';
 import PropTypes from 'prop-types';
 import WaiversGrid from '../../waivers/WaiversGrid';
 import useViewPort from '../../../common/useViewPort';
-import './createEvent.css';
+import styles from './CreateEvent.module.css';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module './CreateEvent.module.css'.